java/src/jmri/jmrit/roster/RosterSpeedProfile.java

```diff
@@ -821,7 +821,7 @@ void calculateStepDetails(float speedStep, float distance, boolean andStop) {
             }
         }
 
-        float endspd = 0;
+        float endspd = minReliableOperatingSpeed;
```

Comment:
The endspd is a speed in mm/sec (0.0->floatmax), the minReliableOperatingSpeed is a speed in percenage throttle (0.0->1.0). So this does nothing meaning full.

Reply:
Pugwash. I hope you agree that endspd should not be 0. I will try to convert minReliableOperatingSpeed to a percentage throttle and see if it has the same desired effect. Hopefully you think that this approach is OK?

Reply:
convert to mm/sec

Reply:
@bill-fitch14 Does not make any diffeence. When this routine is called to desiredSpeedStep and speedStep are set to the minReliableOperatingSpeed. When desiredSpeedStep is not zero the subsequent code uses that to set the endspd. So this code change does nothing. I have no clue what you thought it would do.